delocate/cmd/delocate_wheel.py

```diff
@@ -59,6 +69,15 @@ def main():
     else:
         require_archs = opts.require_archs
     lib_filt_func = 'dylibs-only' if opts.dylibs_only else None
+
+    exclude = [
+        _.strip() for _ in (opts.exclude or '').split(',') if _.strip()]
+
+    def copy_filt(name):
+        if not exclude:
+            return filter_system_libs(name)
+        return filter_system_libs(name) and not _is_in_list(name, exclude)
```

Comment:
I think you are doing a substring search here?  Is that what you intended?  Or did you mean

```
and not name in exclude
```

?

Reply:
The substring search by rossant is intentional.  The main alternative is to allow RegEx, which is usually overkill.